service/history/api/startworkflow/api.go

```diff
@@ -332,24 +332,21 @@ func (s *Starter) verifyNamespaceActive(creationParams *creationParams, currentW
 	return nil
 }
 
-// applyWorkflowIDReusePolicy applies the workflow ID reuse policy in case a workflow start requests fails with a
-// duplicate execution.
-// At the time of this writing, the only possible action here is to terminate the current execution in case the start
-// request's ID reuse policy is TERMINATE_IF_RUNNING.
```

Comment:
I don;t think this comments needs to talk about the internals of `resolveDuplicateWorkflowID`